src/GeositeFramework/js/SubRegion.js

```diff
@@ -146,7 +146,15 @@
         _.each(subregions.areas, function(subregion) {
             var geom = new esri.geometry.Polygon(subregion.shape);
             var symbol = new esri.symbol.SimpleFillSymbol();
+            var outline = new esri.symbol.SimpleLineSymbol(
+                    esri.symbol.SimpleLineSymbol.STYLE_SOLID,
+                    new dojo.Color(subregion.outlineColor || subregions.outlineColor), 2);
```

Comment:
...because the opacity was still set. If the opacity isn't set, here's what it looks like:

![selection_068](https://cloud.githubusercontent.com/assets/1042475/6802908/6badd562-d20a-11e4-95a8-f63be1255626.png)